.palantir/revapi.yml

```diff
@@ -27,3 +27,18 @@ acceptedBreaks:
       new: "method org.apache.thrift.transport.TSocket com.palantir.atlasdb.keyvalue.cassandra.InstrumentedTSocket.Factory::create(java.lang.String,\
         \ int, int)"
       justification: "revert"
+  "0.782.0":
+    com.palantir.atlasdb:atlasdb-api:
+    - code: "java.class.removed"
+      old: "class com.palantir.atlasdb.transaction.service.TransactionStatuses"
+      justification: "Internal methods"
+    - code: "java.class.removed"
+      old: "interface com.palantir.atlasdb.transaction.service.TransactionStatus.Cases<R\
+        \ extends java.lang.Object>"
+      justification: "Internal methods"
+    - code: "java.method.removed"
+      old: "method <R> R com.palantir.atlasdb.transaction.service.TransactionStatus::match(com.palantir.atlasdb.transaction.service.TransactionStatus.Cases<R>)"
+      justification: "Internal methods"
+    - code: "java.method.removed"
+      old: "method boolean com.palantir.atlasdb.transaction.service.TransactionStatus::equals(java.lang.Object)"
+      justification: "Internal methods"
```

Comment:
Are any of these breaks going to impact actual clients? I think these are all just internal AtlasDB APIs but we should confirm.

Reply:
I did not see any imports of these classes when searching our internal repos.